src/EventExport/FileWriter/TabularDataEventFormatter.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace CultuurNet\UDB3\EventExport\FileWriter;
+
+class TabularDataEventFormatter
+{
+    /**
+     * @var string[]
+     */
+    protected $includedProperties;
+
+    protected $columns;
```

Comment:
It's a bit vague what $columns is / should consist of. Maybe it would be better to move this information entirely inside TabularDataEventFormatter? The downside of that would be that TabularDataEventFormatter then probably also gains the responsibility of returning table headers.

Just documenting the structure of the $columns properly might be sufficient for now.